dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py

```diff
@@ -3000,6 +3000,123 @@ def snapshot_page(n_clicks, url, wait_selector):
 
 ])
 
+# # # # # # #
+# Dash App SQL Database Connections
+# # # # # # #
+DataConnections = html.Div(children=[
+    html.H1('Dash App SQL Database Connections'),
+
+    rc.Markdown(
+    '''
+
+    ## Managing Connection Drivers and Libraries
+
+    Dash Apps can use open source libraries in the Python ecosystem to query external databases and datastores in callbacks or job queues.
+
+    The Python DB-API standardizes the interface for most Python database access modules. This allows us to write Dash
+    apps with code that can be modified with little effort to connect to different types of databases with different dialects
+    by following the standardized API. For more information on the Python DB-API, see [PEP-0249](https://www.python.org/dev/peps/pep-0249/).
+
+    To query databases, you'll need to install the Python database packages and in many cases system level dependencies.
+    For complete example, see the "Database Connections" section in Sample Apps & Templates:
+
+    * [Connecting to PostgreSQL](/templates/postgresql-sample-app)
+    * [Connecting to MSSQL](/templates/mssql-pyodbc-sample-app)
+    * [Connecting to MySQL](/templates/pymysql-sample-app)
+    * [Connecting to Oracle Express](/templates/oracle-sample-app)
+    * [Connecting to Databricks Cluster](/templates/databricks-connect)
+    * [Connecting to Dask Cluster](/templates/dask-sample)
+
+    ## Working with Connection Objects in a Dash App
+
+    Many of the database apps follow a similar pattern:
+
+    1. Install the database driver & system level dependencies. This will vary by database type and we recommend viewing
+    the examples that we've provided first. Many of the database drivers more easily installed on Linux environments,
+    and so we recommend developing these apps in a [Dash Enterprise Workspace](/Docs/workspaces).
+
+    2. Create a connection. We recommend using [SQLAlchemy](https://www.sqlalchemy.org) if available for your database.
+
+    3. Store your connection's database password as an environment variable in your App Settings in the Dash Enterprise
+    App Manager instead of storing it within code.
+
+    4. Poll the database initially to check that it is ready to begin accepting connections. We can create a simple
```

Comment:
Instead of using "poll the database" here, I wonder if we could simplify the language a bit to avoid jargon. Also, I think we're actually testing multiple things in this step:

```suggestion
    4. Test that the database server has started successfully, verify that it is accepting connections, and validate the credentials and connection string; we can create a simple
```